lighthouse-core/test/results/sample_v2.json

```diff
@@ -3501,14 +3501,14 @@
         ],
         "items": [
           {
-            "url": "http://localhost:54819/dobetterweb/lighthouse-480x318.jpg",
+            "url": "http://localhost:10200/dobetterweb/lighthouse-480x318.jpg",
             "fromProtocol": true,
-            "isCrossOrigin": true,
+            "isCrossOrigin": false,
             "totalBytes": 24620,
             "wastedBytes": 9028
           }
         ],
-        "overallSavingsMs": 0,
+        "overallSavingsMs": 150,
```

Comment:
ahhhh, do our traces/devtoolsLogs for this use `10200`? if so all these opportunities have been useless as a canary 😅 

much more important fix than we realized!